### PR TITLE
added getTaskTemplate

### LINKED ISSUE #846: Add GetTaskTemplateRequest
### Describe the chore

@DasProffi requested an `GetTaskTemplateRequest` for getting a TaskTemplate by ID

### PR DESCRIPTION
**Pull request by PaulKalho**
## Which issues does this pull request close?
closes #846

### CODE REVIEW

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Can you update `getTaskTemplate()` in `TaskTemplateCRUD_test.go` to use `GetTaskTemplate`. Later on in `TestCreateUpdateGetTaskTemplate()` you can try `GetAllTaskTemplates` so both methods are tested.

File: services/tasks-svc/internal/task-template/task_template.go [hunk 1]
```diff
@@ -324,3 +324,45 @@ func (ServiceServer) GetAllTaskTemplates(ctx context.Context, req *pb.GetAllTask
...
+	if len(rows) == 0 {
+		return nil, status.Error(codes.NotFound, "task template not found")
+	}
+	taskTemplate := rows[0].TaskTemplate
+
+	taskTemplateSubtasks := hwutil.FlatMap(rows, func(row task_template_repo.GetTaskTemplateWithSubtaskByIDRow) **pb.GetTaskTemplateResponse_SubTask {
+		if !row.SubTaskID.Valid {
+			return nil
+		}
+		st := &pb.GetTaskTemplateResponse_SubTask{
```
Comment (FoseFx) [inline services/tasks-svc/internal/task-template/task_template.go:350]: nit: could be an immediate return

**Review by FoseFx — APPROVED** [review]

File: services/tasks-svc/repos/task_template_repo.sql [hunk 2]
```diff
@@ -3,6 +3,19 @@ INSERT INTO task_templates
 	(name, description, created_by, ward_id) VALUES ($1, $2, $3, $4)
     RETURNING id, consistency;
 
+-- name: GetTaskTemplateWithSubtaskByID :many
```
Comment (FoseFx) [inline services/tasks-svc/repos/task_template_repo.sql:6]:
```suggestion
-- name: GetTaskTemplateWithSubtasksByID :many
```

File: services/tasks-svc/stories/TaskTemplateCRUD_test.go [hunk 3]
```diff
@@ -126,4 +120,25 @@ func TestCreateUpdateGetTaskTemplate(t *testing.T) {
...
+	}
+	_, err = taskTemplateClient.CreateTaskTemplate(ctx, createReq)
+	assert.NoError(t, err, "could not create task template")
+	hwtesting.WaitForProjectionsToSettle()
+
+	//
+	// get all templates
+	//
+
+	templates, err := taskTemplateClient.GetAllTaskTemplates(ctx, &pb.GetAllTaskTemplatesRequest{})
```
Comment (FoseFx) [inline services/tasks-svc/stories/TaskTemplateCRUD_test.go:140]: is there anything scoping the task templates response here (e.g., user, orga, ...)? if not this is racy against other tests, which might create task templates

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**